### ISSUE TITLE
[Feature Request] One button cycle through camera presets

### ISSUE THREAD

**Issue by antispamtutoruser123** [praydog/UEVR#70]
I tried assigning some camera presets and it didn't accept my controller input so besides it not seeming to work, 
it would be nice if there was a way to bind one button that cycles through the camera presets.

**Comment by praydog** [comment]
There should be if you hold RT and go through some of the shortcut pages for controllers.

Or do you want something like "Press F1 to go to the next camera preset"?

**Comment by antispamtutoruser123** [comment]
Yeah but not Press F1.  Ideally you could bind it to a controller input so that you have a "camera" button in-game.

If you bind the same camera button to all of your camera presets in UEVR it will just cycle through them when you push the button sequence.
